weakSubjectivityState.json

```diff
@@ -0,0 +1,4 @@
+{
+  "stateRoot": "0x6448d69d56cb5257eb165d20b5e14e65f6b327f76dd76b2dedf80159643207f9",
+  "ipfsPath": "/ipfs/QmVsGxwZs4acTDfhQrNYVqccEVxyhALqJuYqgJrbbNAd1Z/state.ssz"
+}
```

Comment:
Should this and ws-state-root.ssz be ignored?

Reply:
ah man thought i deleted ws-state-root.ssz.  i'll delete it in next commit.  and yeah i'll add this file to .gitignore.